spot-setup/odm/README.md

```diff
@@ -0,0 +1,68 @@
+and finally execute hive query scripts that creates Hive tables needed to access Apache Spot data through the ODM.
+
+## Environment Variables
+
+**spot.conf** is the file storing the variables needed during the installation process including node assignment, User interface, Machine Learning and Ingest gateway nodes.
+This file also contains sources desired to be installed as part of Apache Spot, general paths for HDFS folders, Kerberos information and local paths in the Linux filesystem for the user as well as for machine learning, ipython, lda and ingest processes.
+To read more about these variables, please review the [wiki] (https://github.com/Open-Network-Insight/open-network-insight/wiki/Edit%20Solution%20Configuration).
+
+By default, **odm_setup.sh** expects **spot.conf** to be located in the **/etc** directory on the node. An example spot.conf file is provided in the spot-setup parent directory to help you get started.
+
+## ODM Database Query Scripts
+
+spot-setup/odm contains a script per each table as specified in the Apache Spot Open Data Model (ODM) document.
+These HQL scripts are intended to be executed as a Hive statement and must comply HQL standards.
+
+We want to create tables in Avro/Parquet format to get a faster query performance. This format is an industry standard and you can find more information about it on:
+- Avro is a data serialization system - https://avro.apache.org/
+- Parquet is a columnar storage format - https://parquet.apache.org/
+
+The ODM database query scripts are referenced and executed as part of running the **odm_setup.sh** script.
+Data is meant to be ingested directly into the ODM directories that are tied to the external tables created from the script.
+
+#### ODM Tables
+- event - includes event logs from common data sources used to detect threats and includes network flows, operating system logs, IPS/IDS logs, firewall logs, proxy logs, web logs, DLP logs, etc.
+- user_context - includes information from user and identity management systems.
+- endpoint_context - includes information about endpoint systems (servers, workstations, routers, switches, etc.) and can be sourced from asset management systems, vulnerability scanners, and endpoint management/detection/response systems
+- threat_intelligence_context - includes contextual information about URLs, domains, websites, files and others.
+- network_context - includes information about the network, which can be gleaned from Whois servers, asset databases and other similar data sources.
+- vulnerabilty_context - includes information about vulnerabilities present on endpoint systems that can be sourced from vulnerability scanners and management systems.
+
+## Licensing
+
+spot-setup/odm is licensed under Apache Version 2.0
+
+## Contributing
+
+Create a pull request and contact the maintainers.
+
+## Issues
+
+Report issues at the Apache Spot [issues] (https://github.com/Open-Network-Insight/open-network-insight/issues) page.
```

Comment:
@TaddWood i'm risking being redundant here but this should be changed as well, 
https://issues.apache.org/jira/projects/SPOT/issues

Reply:
@natedogs911 no worries, this one has been updated as well.  Thanks for the heads up.